client/components/Planning/PlanningHistory.jsx

```diff
@@ -127,6 +132,18 @@ export class PlanningHistory extends React.Component {
                                                 </a>
                                             </div>
                                         )}
+                                        {historyItem.operation === PLANNING.HISTORY_OPERATIONS.CREATE_EVENT &&
```

Comment:
This link should also be available if the Planning item was created from an Event